Library/Homebrew/cask/cask.rb

```diff
@@ -386,6 +401,14 @@ def to_hash_with_variations
 
     private
 
+    sig { returns(T.nilable(Homebrew::BundleVersion)) }
+    def bundle_version
+      @bundle_version ||= if (bundle = artifacts.find { |a| a.is_a?(Artifact::App) }&.target) &&
+                             (plist = Pathname("#{bundle}/Contents/Info.plist")) && plist.exist?
+        ::Homebrew::BundleVersion.from_info_plist(plist)
```

Comment:
```suggestion
        Homebrew::BundleVersion.from_info_plist(plist)
```

unless there is a reason for this?